library/Icingadb/Widget/ItemList/BaseDowntimeListItem.php

```diff
@@ -131,24 +132,30 @@ protected function assembleTitle(BaseHtmlElement $title)
             $link = $this->createServiceLink($this->item->service, $this->item->service->host, true);
         }
 
+        if ($this->item->is_flexible) {
+            $template = t('{{#link}}Flexible Downtime{{/link}} for %s');
+        } else {
+            $template = t('{{#link}}Fixed Downtime{{/link}} for %s');
+        }
+
         if ($this->getNoSubjectLink()) {
-            $title->addHtml(new HtmlElement(
-                'span',
-                Attributes::create(['class' => 'subject']),
-                Text::create($this->item->is_flexible ? t('Flexible Downtime') : t('Fixed Downtime'))
+            $title->addHtml(TemplateString::create(
+                $template,
+                [
+                    'link' => HtmlElement::create(
+                        'span',
+                        [ 'class' => 'subject']
+                    )
+                ],
+                $link
             ));
         } else {
-            $title->addHtml(new Link(
-                $this->item->is_flexible
-                    ? t('Flexible Downtime')
-                    : t('Fixed Downtime'),
-                Links::downtime($this->item)
+            $title->addHtml(TemplateString::create(
+                $template,
+                ['link' => new Link('', Links::downtime($this->item))],
+                $link
             ));
         }
-
-        if ($link !== null) {
```

Comment:
You have no alternative for this. If the object link is disabled it looks this way in an object's detail view:
![Screenshot from 2021-07-15 10-25-48](https://user-images.githubusercontent.com/16668527/125755366-00ce83b2-ca83-4db8-bea1-374aa515a687.png)

Reply:
✅